apps/i18n/common/en_us.json

```diff
@@ -515,6 +515,7 @@
   "coteacherRemoveDialogHeader": "Remove {email} as a co-teacher?",
   "coteacherRemoveDialogDescription": "This teacher will lose their ability to manage or view student work for this section.",
   "coteacherNoCoteachers": "You haven't added any co-teachers yet",
+  "coteacherAlreadyInCourse": "{email} is a participant for this course. A user cannot be both a participant and teacher.",
```

Comment:
Should this be "cannot be both a participant and instructor", since the participants are already teachers?